private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -123,7 +123,7 @@ class Dashboard extends React.Component<Props> {
     }
   }
   selectTemplate = (templateID: string) => {
-    this.props.history.push("preview/" + templateID);
+    this.props.history.push("/preview/" + templateID);
```

Comment:
curious, is this a necessary change?

Reply:
yep, it's a necessary change. If there's no slash in front of `preview` than when router does the routing it will use relative address instead of absolute. Example: /template/all => after push becomes /template/preview/..... instead of /preview/...